### ISSUE TITLE
Copyright violations

### ISSUE THREAD

**Issue by YorkAARGH** [CyRIC21/cryptide#1]
Hello CyRIC21, I was linked to this repository on the grounds of you using old "Misaki Code", which was false, you're not running Misaki code, how ever I have noticed you're running [GuideBot](https://github.com/AnIdiotsGuide/guidebot/) code in the following files.

`/events/message.js`
`/util/Logger.js`
`/modules/functions.js`
`/commands/mylevel.js`

you have also copied the entire codebase for [canvas-constructor](https://github.com/kyranet/canvasConstructor/) by myself and @kyranet.

`/src/canvas.js`
`/src/filters.js`

And, you're running MIT licensed code from [Komada](https://github.com/bdistin/OhGodMusicBot/tree/KomadaPackage/OhGodMusicBot) by @bdistin

`/commands/add.js`
`/commands/join.js`
`/commands/play.js`
`/commands/queue.js`

The only reason I'm bringing this to light, is you're not respecting our copyright licenses.

In fact you've gone as far as threatening to take people to court if they use your "All Rights Reserved" code.

The MIT license which [GuideBot](https://github.com/AnIdiotsGuide/guidebot/blob/master/LICENSE), [canvas-constructor](https://github.com/kyranet/canvasConstructor/blob/master/LICENSE) and [OhGodMusicBot Komada Piece](https://github.com/bdistin/OhGodMusicBot/blob/KomadaPackage/OhGodMusicBot/LICENSE.txt) are licensed under, clearly states you must supply our respective copyright notices with the files in question.

![](http://evie.ban-hammered.me/i/cip8m.png)

**Comment by kyranet** [comment]
Good afternoon, dear CyRIC21. I have also noticed uncredited code from [OhGodMusicBot](https://github.com/bdistin/OhGodMusicBot/tree/KomadaPackage)'s KomadaPackage branch, made by @bdistin in your last commit ([`ea51d32`](https://github.com/CyRIC21/cryptide/commit/ea51d326d332ef08504e9484b6e25a2587a329ba)), which is also under the MIT license.

**Comment by CyberiumShadow** [comment]
TL;DR Cease these violations by either adding the respective licenses to your repository or remove the offending code completely

**Comment by KingDGrizzle** [comment]
Hell, for the canvas stuff you could've just installed canvas-constructor, which is on npm!

**Comment by CyRIC21** [comment]
canvas constructor does not work with my hosting provider so i have to use a prebuilt one

**Comment by CyRIC21** [comment]
fixed issues

**Comment by KingDGrizzle** [comment]
Dude. Canvas contructor is not canvas. What you're probably talking about is canvas-prebuilt, a prebuilt version of canvas that's easy to install anywhere. Canvas constructor (and it's related files in `src/`) is a utility npm package that makes canvas be super easy to use.

Also, you didn't "fix the issue". You need to give correct credits in the files that are taken from other sources.

**Comment by CyRIC21** [comment]
ok

**Comment by CyRIC21** [comment]
fixed

**Comment by YorkAARGH** [comment]
Whilst this is certainly a step in the correct direction, you still need to include the licenses as a comment block header for all of the files you utilize.
